tests/integration/tabular/test_base.py

```diff
@@ -38,23 +38,23 @@ def test_conditional_sampling_graceful_reject_sampling_True_dict(model):
         model.sample_conditions(conditions=conditions)
 
 
-#@pytest.mark.parametrize('model', MODELS)
-#def test_conditional_sampling_graceful_reject_sampling_True_dataframe(model):
-#    data = pd.DataFrame({
-#        'column1': list(range(100)),
-#        'column2': list(range(100)),
-#        'column3': list(range(100))
-#    })
-#
-#    model.fit(data)
-#    conditions = pd.DataFrame({
-#        'column1': [28],
-#        'column2': [37],
-#        'column3': [93]
-#    })
-#
-#    with pytest.raises(ValueError):
-#        model.sample(conditions=conditions, graceful_reject_sampling=True)
+@pytest.mark.parametrize('model', MODELS)
+def test_conditional_sampling_graceful_reject_sampling_True_dataframe(model):
+    data = pd.DataFrame({
+        'column1': list(range(100)),
+        'column2': list(range(100)),
+        'column3': list(range(100))
+    })
+
+    model.fit(data)
+    conditions = pd.DataFrame({
+        'column1': [28],
+        'column2': [37],
+        'column3': [93]
+    })
+
+    with pytest.raises(ValueError):
```

Comment:
For almost all the models, I expect the error message 'No valid rows could be generated with the given conditions.', but `GaussianCopula` actually throws a `ValueError: cannot reshape array of size 0 into shape (0)`. I can make a follow-up issue for this, because it seems out of scope to debug this.